target_tests/transfer_size_test.py

```
@@ -0,0 +1,27 @@
+from time import sleep
+
+
+def main():
+    FORMAT = '%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s'
+    logging.basicConfig(format=FORMAT)
+    logging.getLogger().setLevel(logging.INFO)
+
+    with MCUmgrClient.usb_serial("2fe3:.*:2") as client:
+        client.use_auto_frame_size()
+
+        data = str()
```

Ruff (0.16.1)

[warning] Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)